packages/editor/src/components/index.js

```diff
@@ -8,6 +8,7 @@ export { default as BlockEdit } from './block-edit';
 export { default as BlockFormatControls } from './block-format-controls';
 export { default as BlockNavigationDropdown } from './block-navigation/dropdown';
 export { default as BlockIcon } from './block-icon';
+export { default as __experimentalBlockTypesList } from './block-types-list';
```

Comment:
What have you planned in regards to this component?

Reply:
>What have you planned in regards to this component?

The big thing for me is an uncertainty that we'd want to expose `items` as the main prop for `BlockTypesList`, where `items` is currently very bound to the concept of an inserter. It doesn't strike me as appropriate for an interface which otherwise communicates listing... well, any set of block types.